shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml

```diff
@@ -87,13 +87,29 @@ services:
         condition: service_healthy
     ports:
       - "9195:9195"
+      - "8090:8090"
     networks:
       - shenyu
     healthcheck:
       test: [ "CMD", "wget", "http://shenyu-integrated-test-http:9195/actuator/health" ]
       timeout: 2s
       retries: 30
 
+  prometheus:
+    image: prom/prometheus:latest
```

Comment:
We'd better use a fixed version to avoid problems that may occur in the future.

Reply:
ok，I get